src/vit-servicing-station/vit-servicing-station-lib/src/db/queries/snapshot.rs

```diff
@@ -44,6 +44,21 @@ pub fn put_snapshot(snapshot: Snapshot, pool: &DbConnectionPool) -> Result<(), H
     Ok(())
 }
 
+pub async fn query_voters_by_snapshot_tag(
+    tag: String,
+    pool: &DbConnectionPool,
+) -> Result<Vec<Voter>, HandleError> {
+    let db_conn = pool.get().map_err(HandleError::DatabaseError)?;
+    tokio::task::spawn_blocking(move || {
+        voters::dsl::voters
+            .filter(voters::dsl::snapshot_tag.eq(tag))
+            .load(&db_conn)
+            .map_err(|e| HandleError::NotFound(format!("Error loading voters: {}", e)))
```

Comment:
This will have to be changed to support both SQLite and Postgres backends, like:
```suggestion
        crate::q!(db_conn, voters::dsl::voters
            .filter(voters::dsl::snapshot_tag.eq(tag))
            .load(&db_conn))
            .map_err(|e| HandleError::NotFound(format!("Error loading voters: {}", e)))
```